### PR TITLE
fix: add FAQ for VS Code Desktop redirect

### PR DESCRIPTION
**Pull request by loujaybee**
Update FAQ for VS Code Desktop (whether you can open in the same, not a new window). 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3593"></a>

### CODE REVIEW

File: gitpod/docs/references/ides-and-editors/faqs.md [hunk 1]
```diff
@@ -64,3 +64,7 @@ No, JetBrains Remote Development and Gateway are available only in the IntelliJ
 ## Can I use JetBrains Projector with Gitpod?
 
 Gitpod does not officially support [JetBrains Projector](https://lp.jetbrains.com/projector/). Gitpod has support for [JetBrains Gateway](/docs/integrations/jetbrains-gateway), which is the primary tool for remote development when using JetBrains IDE's.
+
+## Can I ensure VS Code Desktop always opens in a single window
```
Comment (loujaybee) [inline gitpod/docs/references/ides-and-editors/faqs.md:68]:
```suggestion
## Can I ensure VS Code Desktop always opens in a single window?
```

File: gitpod/docs/references/ides-and-editors/faqs.md [hunk 2]
```diff
@@ -64,3 +64,7 @@ No, JetBrains Remote Development and Gateway are available only in the IntelliJ
 ## Can I use JetBrains Projector with Gitpod?
 
 Gitpod does not officially support [JetBrains Projector](https://lp.jetbrains.com/projector/). Gitpod has support for [JetBrains Gateway](/docs/integrations/jetbrains-gateway), which is the primary tool for remote development when using JetBrains IDE's.
+
+## Can I ensure VS Code Desktop always opens in a single window?
+
+Unfortunately, it is not possible to redirect to VS Code Desktop using the same window, this is a limitation of VS Code.
```
Comment (loujaybee) [inline gitpod/docs/references/ides-and-editors/faqs.md:70]:
```suggestion

When opening a workspace in VS Code (Desktop) a blank/empty window is opened. A new window is used to open the connection to the workspace and this blank window is left in the background. Unfortunately, it is not possible to redirect to VS Code Desktop using the same window, and/or remove the blank window as this is a limitation of VS Code.
```

File: gitpod/docs/references/ides-and-editors/faqs.md [hunk 3]
```diff
@@ -64,3 +64,8 @@ No, JetBrains Remote Development and Gateway are available only in the IntelliJ
 ## Can I use JetBrains Projector with Gitpod?
 
 Gitpod does not officially support [JetBrains Projector](https://lp.jetbrains.com/projector/). Gitpod has support for [JetBrains Gateway](/docs/integrations/jetbrains-gateway), which is the primary tool for remote development when using JetBrains IDE's.
+
+## Can I ensure VS Code Desktop always opens in a single window?
+
+
+When opening a workspace in VS Code (Desktop) a blank/empty window is opened. A new window is used to open the connection to the workspace and this blank window is left in the background. Unfortunately, it is not possible to redirect to VS Code Desktop using the same window, and/or remove the blank window as this is a limitation of VS Code.
```
Comment (loujaybee) [inline gitpod/docs/references/ides-and-editors/faqs.md:71]: I re-used some of the language from this issue: 

https://github.com/gitpod-io/gitpod/issues/7527

Hopefully that makes it easier for user to search or find, as it's in their words...

**Review by Siddhant-K-code — APPROVED** [review]

File: gitpod/docs/references/ides-and-editors/faqs.md (hunk 3 above)
Comment (filiptronicek) [reply gitpod/docs/references/ides-and-editors/faqs.md:71]: We can specify that this happens only when the user does not have vsc opened already?

Comment (filiptronicek) [inline gitpod/docs/references/ides-and-editors/faqs.md:71]:
```suggestion
When opening a workspace in VS Code (Desktop) without it being already open a blank/empty window is opened first. A new window is then used to open the connection to the workspace and this blank window is left in the background. Unfortunately, it is not possible to redirect to VS Code Desktop using the same window, and/or remove the blank window as this is a limitation of VS Code.
```

**Review by filiptronicek — APPROVED** [review]

**Comment by Siddhant-K-code** [comment]
This looks good to go 🚀. :shipit:

**Comment by filiptronicek** [comment]
Actually, we could mention that now we try to reuse windows when they're blank: https://github.com/gitpod-io/gitpod-vscode-desktop/commit/829a8150df2247be3cca4a3b47477a18339b188c

**Review by filiptronicek — COMMENTED** [review]
Actually, we could mention that now we try to reuse windows when they're blank: https://github.com/gitpod-io/gitpod-vscode-desktop/commit/829a8150df2247be3cca4a3b47477a18339b188c

**Pull request closed without merging**